pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
@olegshaldybin The assumption is that only cluster-admins can define roles.  

The covers check is done in the validation rather than in an admission controller.  I assume this is because you would not ever want to make it optional.  Is that right @liggitt?  Asking about doing covers check in validation vs admission.